### DISCUSSION TITLE
How do we update or upsert tables joined with foreign keys?

### DISCUSSION THREAD

**Discussion by fergusmeiklejohn** [supabase/supabase#6771] [Questions] [answered]
This question was asked in a thread 9 months ago: https://github.com/supabase/supabase/discussions/710#discussioncomment-1145583

Can we perform an update or upsert on multiple tables joined with foreign keys using the supabase JS client?

If that's impossible we could use transactions as an alternative

**Comment by steve-chavez** [comment, marked as answer]
Not yet, but this is being tracked on https://github.com/supabase/postgrest-js/issues/240.
